challenge-13/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,199 @@
+		}
+		return nil, err
+	}
+
+	return p, nil
+}
+
+// UpdateProduct updates an existing product
+func (ps *ProductStore) UpdateProduct(product *Product) error {
+	// TODO: Update the product in the database
+	// TODO: Return an error if the product doesn't exist
+	_, err := ps.db.Exec("UPDATE products SET name = ?, price = ?, quantity = ?, category = ? WHERE id = ?",
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)